File: 2037. Minimum Number of Moves to Seat Everyone/solution.cpp
class Solution {
public:
  int minMovesToSeat(vector<int>& seats, vector<int>& students) {
    vector<int> sortedSeats = seats;
    sort(sortedSeats.begin(), sortedSeats.end());
    vector<int> sortedStudents = students;
    sort(sortedStudents.begin(), sortedStudents.end());

    int result = 0;

    for (int index = 0; index < seats.size(); index++){
      int movements = abs(sortedSeats[index] - sortedStudents[index]);
      result += movements;
    }

    return result;
  }
};

File: 2037. Minimum Number of Moves to Seat Everyone/solution.java
class Solution {
  public int minMovesToSeat(int[] seats, int[] students) {
    int[] sortedSeats = seats.clone();
    Arrays.sort(sortedSeats);
    int[] sortedStudents = students.clone();
    Arrays.sort(sortedStudents);

    int result = 0;

    for (int index = 0; index < seats.length; index++){
      int movements = Math.abs(sortedSeats[index] - sortedStudents[index]);
      result += movements;
    }

    return result;
  }
}